jobs/release-monitoring/branch/fuse-online-next.yaml

```diff
@@ -0,0 +1,28 @@
+---
+- job:
+    name: release-monitoring-branch/fuse-online
```

Comment:
ERROR: No item named release-monitoring-branch/fuse-online-next found

Reply:
Needs the next on the end, the job name should match the branch name exactly (fuse-online-next)